src/lib/i18n/de.mnestix.ts

```diff
@@ -20,6 +20,8 @@ export const deMnestix = {
     idStructure: 'ID Struktur',
     idStructureExplanation:
         'Definieren Sie, wie Ihre IDs aussehen sollen. Dies ist eine Basis-Einstellung, die für individuelle Importe angepasst werden kann.',
+    mnestixConnections: 'Mnestix Quellen',
+    mnestixConnectionsExplanation: 'Definieren Sie, welche Daten Quellen verwendet werden sollen.',
```

Comment:
Not related to this PR, but do we really want to use the formal "Sie" instead of just "Definiere, welche [...]"?